setup.sh

```diff
@@ -247,34 +287,17 @@ openCRVSPorts=( 3447 9200 27017 6379 8086 4444 3040 5050 2020 7070 9090 1050 303
 for x in "${openCRVSPorts[@]}"
 do
    :
-    if lsof -i:$x; then
-      echo -e "OpenCRVS thinks that port: $x is in use by another application.\r"
+    if lsof -nP -iTCP:$x -sTCP:LISTEN -iUDP:$x >/dev/null; then
```

Comment:
This makes it so that only ports that are listened to either with TCP or UDP protocols are considered. `lsof -i` also lists for instance Robo 3T as it's just trying to open a connection to the MongoDB port